soap/views.py

```
@@ -0,0 +1,92 @@
+
+
+class SOAPFormatsView(APIView):
+    """
+    API برای تولید فرمت‌های مختلف گزارش SOAP
+    """
+
+    permission_classes = [IsAuthenticated, IsDoctor]
+
+    @with_api_ingress(rate_limit=50, rate_window=60)
+    def get(self, request, report_id: int, *args, **kwargs):
+        ingress = APIIngressCore()
```

Ruff (0.12.2)

Local variable `ingress` is assigned to but never used

Remove assignment to unused variable `ingress`

(F841)